docs/releases/1.23-NOTES.md

```diff
@@ -12,6 +12,32 @@ managed subnets will be configured to launch instances with Resource Based Names
 
 # Breaking changes
 
+## Custom domain names on AWS
+
+If your EC2 instances runs in a VPC that has custom domain name set in its DHCP options or custom resolve configuarion, instances will not be able to join the cluster. The reason for this is that the AWS
+cloud provider requires that instances use the default regional domain names. Before kOps 1.23, kOps would fetch the expected node name from the AWS EC2 API, but this
+caused scaling issues on large clusters and kOps now rely on the DNS configuration instead.
+
+As of Kubernetes 1.22, one can enable the [external cloud provider](https://cloud-provider-aws.sigs.k8s.io/), which supports Resource Based Naming.
+When the external cloud provider is used, kOps will automatically use instance IDs as node names instead of IP address based naming. These node
```

Comment:
```suggestion
When the external cloud provider is used, kOps will automatically use instance IDs as node names instead of IP address-based naming. These node
```